test/metabase/api/user_test.clj

```diff
@@ -347,6 +347,31 @@
       :is_superuser true})
     (superuser-and-admin-pgm-info email)))
 
+(deftest create-user-mixed-case-email
+  (testing "POST /api/user/:id"
+    (testing "can create a new User with a mixed case email and the email is normalized to lower case"
+      (let [user-name (random-name)
+            email     (tu/random-email)]
+        (is (= email
+              (:email (et/with-fake-inbox
+                   (try
+                     (tu/boolean-ids-and-timestamps
+                      ((mt/user->client :crowberto) :post 200 "user"
+                       {:first_name       user-name
+                        :last_name        user-name
+                        :email            (u/upper-case-en email)
+                        :login_attributes {:test "value"}}))
+                     (finally
+                       ;; clean up after ourselves
+                       (db/delete! User :email email)))))))))
+
+    (testing "attempting to create a new user with an email with case mutations of an existing email should fail"
+      (is (= {:errors {:email "Email address already in use."}}
+            ((mt/user->client :crowberto) :post 400 "user"
```

Comment:
Similarly since `=` is a function call if its arguments are on separate lines their indentation should match e.g.

```clj
(= x
   y)
```

Indenting all this stuff yourself is a little tricky but a good editor should be able to do it for you. I'm not sure how to have Vim Fireplace do it but @walterl could probably point you in the right direction

Reply:
I just use the Clojure syntax highlighting and indentation rules from [vim-polyglot](https://github.com/sheerun/vim-polyglot), which uses [vim-clojure-static](https://github.com/guns/vim-clojure-static).

Reply:
Thanks for the tip about vim-polyglot!